tests/test_config.py

```diff
@@ -186,31 +186,28 @@ def test_parse_errors(self):
             with pytest.raises(CoverageException, match=msg):
                 coverage.Coverage()
 
-    def test_toml_parse_errors(self):
+    @pytest.mark.parametrize("bad_config,msg", [
+        ("[tool.coverage.run]\ntimid = \"maybe?\"\n", r"maybe[?]"),
+        ("[tool.coverage.run\n", r"Key group"),
```

Comment:
```suggestion
        ("[tool.coverage.run\n", r'Expected "]"'),
```
Seems like something like this may help.

This is asserting an error message from an external library though, which I'm not sure is a great idea. So maybe just go with this instead
```suggestion
        ("[tool.coverage.run\n", None),
```